procrastinate/worker.py

```diff
@@ -169,7 +185,11 @@ def load_task(self, task_name) -> tasks.Task:
             end_time = job_context["end_timestamp"] = time.time()
             job_context["duration_seconds"] = end_time - start_time
             extra = {"action": log_action, **log_context, "result": task_result}
-            self.logger.log(log_level, log_title, extra=extra, exc_info=exc_info)
+            ftext = (
+                f"{log_title} on job {job_context['call_string']} "
```

Comment:
so that may create a log saying `Job error, to retry on job task(...)` and "to retry on job" feels strange.

Maybe it would seem more natural with:
```suggestion
                f"{log_title} - Job: {job_context['call_string']} "
```